types/telegram-web-app/index.d.ts

```diff
@@ -96,11 +100,16 @@ interface WebApp {
      * header of the Web App in the Telegram interface.
      */
     BackButton: BackButton;
+    /**
+     * An object for controlling the secondary button,
+     * which is displayed at the bottom of the Mini App in the Telegram interface.
+     */
+    MainButton: BottomButton;
```

Comment:
Please fix the docs. The main button is documented as secondary button and vice-versa now.